### PR TITLE
Move fs.microsoft.com to Font Streaming Area

### PR DESCRIPTION
**Pull request by Git-0nga**
Font Streaming Update;
Cortana and Live Tiles from Cortana and Search

### CODE REVIEW

**Comment by linque1** [comment]
#sign-off

**Comment by e0i** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: windows/privacy/manage-windows-2004-endpoints.md [hunk 1]
```diff
@@ -64,6 +64,8 @@ The following methodology was used to derive these network endpoints:
 |||TLSv1.2|v20.events.data.microsoft.com|
 ||The following endpoints are used by Windows Error Reporting. To turn off traffic for these endpoints, enable the following Group Policy: Administrative Templates > Windows Components > Windows Error Reporting > Disable Windows Error Reporting. This means error reporting information will not be sent back to Microsoft.|HTTPS|*.telecommand.telemetry.microsoft.com|
 |||TLS v1.2|watson.*.microsoft.com|
+|Font Streaming|||[Learn how to turn off traffic to all of the following endpoint(s).](manage-connections-from-windows-operating-system-components-to-microsoft-services.md#6-font-streaming)|
+||The following endpoints are used to download fonts on demand. If you turn off traffic for these endpoints, you will not be able to download fonts on demand. .|HTTPS|fs.microsoft.com*|
```
Comment (JohanFreelancer9) [inline windows/privacy/manage-windows-2004-endpoints.md:68]:
```suggestion
||The following endpoints are used to download fonts on demand. If you turn off traffic for these endpoints, you will not be able to download fonts on demand. |HTTPS|fs.microsoft.com*|
```

**Comment by e0i** [comment]
@denisebmsft The PR has been approved and can be merged.

**Pull request merged**